src/main/app/case/to-api-format.ts

```diff
@@ -67,8 +67,8 @@ const fields: ToApiConverters = {
   iWantToHavePapersServedAnotherWay: data => ({
     applicant1WantsToHavePapersServedAnotherWay: checkboxConverter(data.iWantToHavePapersServedAnotherWay),
   }),
-  legalProceedingsRelated: data => ({
-    legalProceedingsRelated: data.legalProceedings === YesOrNo.YES ? data.legalProceedingsRelated : [],
+  legalProceedingsByCase: data => ({
+    legalProceedingsByCase: data.legalProceedings === YesOrNo.YES ? data.legalProceedingsByCase : [],
```

Comment:
as above, even if you change the name from the in the content, its really an array of legalProceedingsRelated so the return object is incorrect, itll have to be modified here or parser might need to be created where the data is converted.